src/envs/arithmetic.py

```diff
@@ -211,7 +259,10 @@ def register_args(parser):
         Register environment parameters.
         """
         parser.add_argument(
-            "--operation", type=str, default="gcd", help="Operation to perform"
+            "--operation", type=str, default="data", help="Operation to perform"
```

Comment:
@f-charton I am unsure if you agree with this change, if so, I am happy to reverse it.